Makefile

```diff
@@ -29,17 +31,18 @@ project-build-proxy:
 	cp -f \
 		$(PROJECT_DIR)/certificate/* \
 		$(DOCKER_DIR)/proxy/assets/certificate
+	cp -f -r \
```

Comment:
Can we convert this line to use `cp -rf` so the flags are combined in one set of options?

Reply:
Sure